src/store/auth/slice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const authSlice = createSlice({
+  name: "auth",
+  initialState: {},
```

Comment:
Change the **initialState** to: 
`{
  user: {}
}`

_(and update selectors from `selectors.js`)_